test/geometry/epipolar/test_fundamental.py

```diff
@@ -162,6 +162,70 @@ def test_opencv_svd(self, device, dtype):
         F_mat = epi.find_fundamental(points1, points2, weights)
         assert_close(F_mat, Fm_expected, rtol=1e-4, atol=1e-4)
 
+    def test_8point_opencv(self, device, dtype):
```

Comment:
I'm missing a test for the 7point. Would be also great to test minimal properties from the obtained fundamental matrix. Like `p_left.T @ Fm @ p_right == 0`
https://www.cse.psu.edu/~rtc12/CSE486/lecture19.pdf

Reply:
I have added one test which satisfies the above property. Basically, the same test used for testing 8 point algorithm ([https://github.com/kornia/kornia/blob/master/test/geometry/epipolar/test_fundamental.py#L166C9-L166C31](url) )

Reply:
but still gradcheck is failing seems torch.linalg.svd  is quite unstable